packages/nc-gui/pages/index/index/create.vue

```diff
@@ -48,6 +52,7 @@ const createProject = async () => {
   } catch (e: any) {
     message.error(await extractSdkResponseErrorMsg(e))
   }
+  creating.value = false
```

Comment:
even though this line is working fine, I'd prefer to put it in `finally` block as well since it starts in `try` block.